django/contrib/admin/templates/admin/submit_line.html

```diff
@@ -1,7 +1,7 @@
 {% load i18n admin_urls %}
 <div class="submit-row">
 {% if show_save %}<input type="submit" value="{% trans 'Save' %}" class="default" name="_save" />{% endif %}
-{% if show_delete_link %}
+{% if show_delete_link and original %}
```

Comment:
It looks like this change is unnecessary - if `show_delete_link` is `True` then `original` will always be set

Reply:
That doesn't seem to be true. When I removed the `original` I got several warnings from original being `None`.